src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -239,20 +239,63 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         }
     }
 
+    /**
+     * Returns a value indicating whether the current asset is taggable
+     */
+    private isTaggableAssetType = (asset: IAsset): boolean => {
+        return asset.type === AssetType.Image || asset.type === AssetType.VideoFrame;
+    }
+
     /**
      * Raised when the selected asset has been changed.
      * This can either be a parent or child asset
      */
     private onAssetMetadataChanged = async (assetMetadata: IAssetMetadata): Promise<void> => {
```

Comment:
this method is still really confusing to me :(

Reply:
@mydiemho the purpose of it or the actual implementation?

Reply:
@mydiemho - Let's discuss offline.